### DISCUSSION TITLE
Share suggestions on default dashboards and templates

### DISCUSSION THREAD

**Discussion by ankit01-oss** [SigNoz/signoz#1695] [Ideas]
We are working on making SigNoz easier to use. In that direction, we are gathering requirements on default dashboards and templates that a user might need to get started quickly with SigNoz.

Examples of default dashboards:
- host metrics dashboard
- Kubernetes dashboard

We are also thinking of creating templates. For example, a template for APM alerts. Let us know what dashboards you usually use for your application and infrastructure, and we will include it in the list.

**Comment by ayush-rathore-quartic** [comment]
Kubernetes Dashboard: Something we had to modify in the default dashboard and collector configuration was to add container-level resource metrics, I think that can be a wider use case if you are working with several multi container pods in your cluster.
